.github/workflows/tests.yml

```diff
@@ -172,36 +174,28 @@ jobs:
           # (sometimes, the install appears to be successful but shared
           # libraries cannot be loaded at runtime, so an actual import is a
           # better check).
-          # PyGObject, pycairo, and cariocffi do not install on OSX 10.12.
-           python -m pip install --upgrade pycairo 'cairocffi>=0.8' PyGObject &&
+          python -m pip install --upgrade pycairo 'cairocffi>=0.8' &&
+            echo 'Cairo is available' ||
+            echo 'Cairo is not available'
+
+          # PyGObject does not install on OSX 11.6.
+          python -m pip install --upgrade PyGObject &&
              python -c 'import gi; gi.require_version("Gtk", "3.0"); from gi.repository import Gtk' &&
              echo 'PyGObject is available' ||
              echo 'PyGObject is not available'
 
-          # There are no functioning wheels available for OSX 10.12 (as of
-          # Sept 2020) for either pyqt5 (there are only wheels for 10.13+) or
-          # pyside2 (the latest version (5.13.2) with 10.12 wheels has a
-          # fatal to us bug, it was fixed in 5.14.0 which has 10.13 wheels)
           python -mpip install --upgrade pyqt5${{ matrix.pyqt5-ver }} &&
             python -c 'import PyQt5.QtCore' &&
             echo 'PyQt5 is available' ||
             echo 'PyQt5 is not available'
-          if [[ "${{ runner.os }}" != 'macOS' ]]; then
-            python -mpip install --upgrade pyside2 &&
```

Comment:
I was also thinking about that. It may indeed be the case, but that OSX should use QT6 if possible is what I took away from the comment https://github.com/matplotlib/matplotlib/pull/23584#discussion_r941803484

Reply:
I think we want both installed.  In the interactive backend tests we push a whole lot of stuff into sub-processes so we can test multiple GUI frameworks and even have a test that we warn if the user mixes Qt5 and Qt6.

Reply:
I pushed a commit to put this back.

Reply:
Ahh, I realize now that I read the if-statement incorrectly... I though it said `== 'macOS'`. So my logic was that since we can install qt6 now on macOS, no need for qt2...